.github/workflows/Build_master.yml

```
@@ -0,0 +1,45 @@
+  build:
+    strategy:
+      matrix:
+        # Java LTS & minimum supported by Minecraft
+        java: [ 17, 23 ]
+        # Run on both Linux and Windows
+        os: [ ubuntu-22.04, windows-2022 ]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout repository (检出代码库)
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)